### PR TITLE
Markdown added and code illustrations

### PR DESCRIPTION
**Pull request by cockochan**
Thank you for contributing! To help the review process, please provide the following:

### Proposal
This PR adds markdown to make questions and css style to display question code illustration as code illustration

### Related
Provide links to any associated pull requests or issues. You can use `#123` to link to a PR or issue in this repository, or `user/repo#123` to link to other repositories.

### Checklist

- [ ] I have read the [code of conduct] and [contributing guide]
- [ ] I have made this pull request to the `master` branch
- [ ] I have run all of the automated validation using `npm run ship`
- [ ] I have added myself to the `"contributors"` list in the `package.json` (or do not want to)

[code of conduct]: https://github.com/textbook/starter-kit/blob/master/.github/CODE_OF_CONDUCT.md
[contributing guide]: https://github.com/textbook/starter-kit/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: client/src/Components/Mentors.js [hunk 1]
```diff
@@ -72,12 +91,17 @@ export default function Mentors(props) {
 	if (props.questions) {
 		return (
 			<div className="row">
+				<div className='filterButtons col-6'>
+					{tagsCollection.map((tag)=>{
+						return(<button>{tag}</button>);
+					})}
+				</div>
```
Comment (naderakhgari) [inline client/src/Components/Mentors.js:98]: can you please add key to fix the warning

File: client/src/Components/Mentors.js [hunk 2]
```diff
@@ -5,12 +5,31 @@ import "../grid.css";
...
+						null;
+					}
+
+				});
+				setTagsCollection(tempTags);
+			});
+
+		}
+
+	};
```
Comment (naderakhgari) [inline client/src/Components/Mentors.js]: can you please remove the line spaces and format the code to be looks nice

File: client/src/App.css [hunk 3]
```diff
@@ -29,7 +30,10 @@
 .newQuiz {
   border: 2px solid black;
 }
-
+.addQuestionButton{
+  margin-left:1.5vw;
+  
+}
```
Comment (naderakhgari) [inline client/src/App.css:36]: you can get rid of line spaces

**Review by naderakhgari — CHANGES_REQUESTED** [review]
just needs some small changes

File: client/src/Components/Mentors.js (hunk 1 above)
Comment (cockochan) [reply client/src/Components/Mentors.js:98]: I have new code waiting to be pushed will fix in next iteration. Unfortunately there are more changes i dont want to commit just now

**Pull request merged**